city-nocache.html

```diff
@@ -7,6 +7,8 @@ <h1>Browse by City</h1>
   </div>
   <div class="row">
     <div class='col-md-6'>
+      <p> Select one of the cities below to discover news reports on Wilde's trials published from that city. The number next to each city indicates the number of reports published in that city. </p> 
+      <div data-template="app:browse-date"/> </div></div>
```

Comment:
This doesn't look right to me. I think you have one too many end div tags there. Also, are you sure you want to include the list of dates in the Browse by City page?